packages/docs/next-doc-site/pages/_document.tsx

```diff
@@ -27,21 +27,21 @@ class CustomDocument extends Document<DocumentInitialProps> {
           <link
             as="font"
             crossOrigin="anonymous"
-            href="https://waffles.datacamp.com/fonts/StudioFeixenSans-Regular.woff2"
+            href="/fonts-split/StudioFeixenSansRegular-english.woff2"
```

Comment:
Do we know how this will affect people's names? A couple examples 
<img width="469" alt="Screen Shot 2021-07-02 at 11 22 39 AM" src="https://user-images.githubusercontent.com/283611/124303544-363bde00-db28-11eb-8ac0-35c3cc77d101.png">
<img width="591" alt="Screen Shot 2021-07-02 at 11 24 52 AM" src="https://user-images.githubusercontent.com/283611/124303549-363bde00-db28-11eb-89f4-a5adf963af70.png">

Reply:
@dandenney It's pretty smart how it works - when it sees some characters from different unicode range (those nordic/slavic characters in this case) it downloads appropriate character set. So when the website loads it will download not only english (_StudioFeixenSansBold-english.woff2_) but also latin extended (_StudioFeixenSansBold-latin-extended.woff2_).

Reply:
Will make knowledge share about it next week. It's kind of interesting topic.